### DISCUSSION TITLE
How would you implement “add on select”  feature of Django?

### DISCUSSION THREAD

**Discussion by zelid** [refinedev/refine#2533] [Ideas]
Thanks for an amazing framework!

How would you implement “add on select”  feature of Django?

Example of “add on select” feature:

1) open https://djangosuit.com/admin/examples/kitchensink/add/
2) click on a plus sign near a country field
3) create country inside opened “country create” page or pop up
4) you get newly created country selected in drop-down (where the plus sign was clicked on 2) step

It looks like a big usability buster.

Has anyone found any examples like this in Refine?

Thanks!

**Comment by omeraplak** [comment]
Hey @zelid 👋,
Thank you for contacting us! We'll create an example next week 🚀

**Reply by zelid** [reply]
Wow, amazing! That was always Django's "killer-feature" for me. Waiting forward to see how to implement it using Refine :)

**Comment by omeraplak** [comment]
Hey @zelid ,
We've created an example for you! 🎁 You can check it out from here -> https://stackblitz.com/edit/pankod-refine-ouyk2b?file=src%2Fpages%2Fposts%2Fcreate.tsx&preset=node

**Reply by zelid** [reply]
Great, thanks a lot for your help!
